services/jobs/src/main/java/com/linkedin/openhouse/jobs/model/JobConf.java

```diff
@@ -17,6 +17,7 @@
 public class JobConf {
   private JobType jobType;
   private String proxyUser;
+  private String memory;
```

Comment:
We might need to put a bit more thoughts into this. 
having memory, which is a very specific param sitting along with `jobType` and `proxyUser` is a bit strange.  I was thinking a more flexible constructs like `executionConfig` which can carry both executor memory and driver memory, for spark jobs. 


cc @teamurko  let's discuss this more. Context: we are considering opening job-service to platform teams if they are managing some of the jobs themselves.

Reply:
`memory` is a job level config hence seemed suitable to put along with `jobType` and `proxyUser`.
I'm okay calling it `memoryConfig` but `executionConfig` seems to hold something more than memory.

Also, since in most cases driver and executor run with same memory, providing params for both is a stretch?